serde-generate/runtime/typescript/bincode/bincodeDeserializer.ts

```diff
@@ -1,8 +1,8 @@
 import {BinaryDeserializer} from "../serde/binaryDeserializer";
 
 export class BincodeDeserializer extends BinaryDeserializer {
-    deserializeLen(): BigInt {
```

Comment:
I guess `Number` is indeed large enough for the maximal length authorized by the BCS specs (or supported by Java vectors anyway).

Reply:
It better be - it's being coerced back to `number` elsewhere! :D